reconcile/openshift_groups.py

```diff
@@ -111,6 +115,14 @@ def fetch_desired_state(oc_map):
     desired_state = []
 
     for r in roles:
+        gql.get_api
+        checkExpirationDate = openshift_resource.checkExpirationDate(r)
+        if checkExpirationDate is True:
+            continue
+        elif checkExpirationDate is False:
+            raise ValueError(
+                f'The maximum expiration date of {r["name"]} '
+                f'shall not exceed {EXPIRATION_MAX} days from today')
```

Comment:
Also, this is shorter and easier:
```suggestion
        if  not openshift_resource.check_expiration_date(r):
            raise ValueError(
                f'The maximum expiration date of {r["name"]} '
                f'shall not exceed {EXPIRATION_MAX} days from today')
```

Reply:
Thank you! Went ahead and fixed that.